src/models/user.ts

```diff
@@ -46,4 +49,15 @@ const Profile = mongoose.model(
   })
 );
 
-export { User, Profile };
+const UserRole = mongoose.model(
+  "UserRole",
+  new Schema({
+    userRole: {
```

Comment:
The field name can not match the model name.

The suggestion into this model would be 

```js
name: string
scope or grants: [{method:[get,post], attributes:[resources: name, image,*]}]
```
### Example 
This would help us filter the data based on the user role too.
```js
name: admin
permissions: [grants]
```

Reply:
Feedback applied.